packages/babel-parser/src/plugins/flow.js

```diff
@@ -1630,7 +1631,7 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         this.match(tt.name) &&
         (this.state.value === "type" ||
           this.state.value === "interface" ||
-          this.state.value == "opaque")
+          this.state.value === "opaque")
```

Comment:
We should enable the eslint rule to only allow `==` for null/undefined.

Reply:
👍